capstone/backend/src/main/java/com/google/spanner/LibraryFunctions.java

```diff
@@ -128,4 +128,8 @@ public static void createUser(int userId, String userName, String email) throws
     }
     return itemTypes;
   }
-}
\ No newline at end of file
+
```

Comment:
If you delete changes in this file, you can submit this. We don't submit code that can't compile, because all tests should pass and someone else should be able to run this file after you submit.
Maybe add a TODO here instead, like:
```
// TODO(bzallen): Implement getStoresWithItems function.
```